api/client/build.go

```diff
@@ -211,15 +211,12 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		}
 	}
 
-	var shmSize int64 = 67108864 // initial SHM size is 64MB
+	shmSize := runconfig.DefaultSHMSize
```

Comment:
Wondering, shouldn't this be empty on the client when not set and set the default on the daemon side?